src/main/drivers/vtx_smartaudio.c

```diff
@@ -0,0 +1,706 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include <ctype.h>
+
+#include "platform.h"
+
+#ifdef VTX_SMARTAUDIO
+
+#include "drivers/system.h"
+#include "drivers/serial.h"
+#include "drivers/vtx_smartaudio.h"
+#include "io/serial.h"
+#include "fc/runtime_config.h"
+#include "config/config_master.h"
```

Comment:
The thing that calls `smartAudioInit` should open the port and pass it in as a parameter to the `smartAudioInit` function. So yes, `main.c`. [We might elect to change that in future, but for the moment that is the correct place.]